pkg/api/upgradelog/handler.go

```diff
@@ -30,7 +30,7 @@ const (
 	defaultJobBackoffLimit int32 = 5
 	logPackagingScript           = `
 #!/usr/bin/env sh
-set -ex
+set -x
```

Comment:
may not be a good idea to remove `-e`, it will hide other potential errors

to escape and check following error, it could be:

`awk '{$1=$2=""; print $0}' $f | jq ... || ret=$?`

then check `ret`, you will know if error happens in `awk & jq`

Reply:
Agreed on the `-e` part. But for the `ret` part, I think it's more simple and clear to keep the original style since using `||` is a short-circuit. If the previous command is executed successfully, there will be no assignment of `ret` variable, hence causing trouble to the following `if` statement.

Reply:
You're right, catching the return code after `||` is necessary since the error-prone command is the last one in the pipeline. I've updated the code, please take a look. Thanks!